jetty-ee10/jetty-ee10-quickstart/src/main/java/org/eclipse/jetty/ee10/quickstart/QuickStartConfiguration.java

```diff
@@ -99,8 +100,9 @@ public void preConfigure(WebAppContext context) throws Exception
             throw new IllegalStateException("Bad Quickstart location");
 
         //look for quickstart-web.xml in WEB-INF of webapp
-        Resource quickStartWebXml = getQuickStartWebXml(context);
-        LOG.debug("quickStartWebXml={} exists={}", quickStartWebXml, quickStartWebXml.exists());
+        Path quickStartWebXml = getQuickStartWebXml(context);
+        if (LOG.isDebugEnabled())
+            LOG.debug("quickStartWebXml={}", quickStartWebXml);
```

Comment:
I think you should keep the `exists={}` log, doing it on the path.

Reply:
Fixed